### ISSUE TITLE
Passing arguments to commands.

### ISSUE THREAD

**Issue by hewhocannotbetamed** [houmain/keymapper#102]
`  switch_systeminformer  >> $(start wscript.exe C:\lazysys\scripts\powershell\utilities\window-management\switch.vbs 'System Informer')
`

This should, in my opinion just-work™. Alas, it does not. I've written powershell and batch scripts, but 2slow. wscript.exe + Windows API shenanigans seems to be the fastest means of "activating" a window per its title. 

If you have another solution you use, feel free to let me know. I'm also curious if/how I can just feed whatever command I want to invoke arguments. On Windows.

**Comment by ristomatti** [comment]
I think this should be reopened. I've been thinking about making a feature request about this. My current workaround is to define part of the command as an alias but it's somewhat of an ugly hack. E.g. I do this:

```
notify = notify-send -t 5000 keymapper

Control{A} >> $(notify 'some notification')

# or
Control{B} >> do_something
do_something >> $(notify 'did something')
```

It would be nicer to be able to do something like this but AFAIK the config is not valid.

```
Control{B} >> do_something 'did something'
```

**Comment by houmain** [comment]
Thank you for the feature requests! They all sound good. I think the discussions area would have been the right place for this list. Maybe you could create a topic there.
- I think I could make the virtual modifier state accessible using DBUS.
- Which external data are you thinking of? Arguments for shell commands?
- I have already thought about the third point.

**Comment by ristomatti** [comment]
Good point, I've replied here https://github.com/houmain/keymapper/discussions/103.

**Comment by houmain** [comment]
>  Maybe you can give me your script so I can try it out myself?

No longer needed. I found the problem and fixed it in 3.5.2.
